apps/web/src/middleware.ts

```
@@ -4,6 +4,28 @@ import { isAuthenticated, refreshNewAccessToken } from '~/app/_common/actions/au
 import { unauthenticatedRoutes } from '~/path';
 
 export async function middleware(request: NextRequest) {
+  const host = request.headers.get('host');
+  const mainDomain = process.env.NEXT_PUBLIC_DOMAIN!;
+
+  if (
+    host &&
+    host.includes('.') &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)